874816 : UG Documentation for PdfRemoteGoToAction support.

AntroJames opened the pull request:
### Description : 
Updated UG documentation content for PdfRemoteGoToAction support.

SyncfusionBuild commented:
**Build Status: INQUEUE :clock3:<br/>Build Location: [Task_874816](https://ci.syncfusion.com/job/Content/job/fileformat-docs/54/console)**

SyncfusionBuild commented:
**CI Status: FAILURE :x:<br/>Source Compilation: SUCCESS :white_check_mark:<br/>Failure Reason: [CI Failure with violation](https://ci.syncfusion.com/job/Content/job/fileformat-docs/54/artifact/cireports/)<br/>Build Location: [Task_874816](https://ci.syncfusion.com/job/Content/job/fileformat-docs/54/)<br/>Image Alt Text Error(s): [0](https://ci.syncfusion.com/job/Content/job/fileformat-docs/54/artifact/cireports/ImageValidation/AltTextValidation.html)<br/>Image Size Error(s): [0](https://ci.syncfusion.com/job/Content/job/fileformat-docs/54/artifact/cireports/ImageValidation/ImageSizeValidation.html)<br/>Image Name Error(s): [0](https://ci.syncfusion.com/job/Content/job/fileformat-docs/54/artifact/cireports/ImageValidation/ImageNameValidation.html)<br/>FT Layout Syntax Error(s): [0](https://ci.syncfusion.com/job/Content/job/fileformat-docs/54/artifact/cireports/FTLayoutSyntaxValidation/FTStructureValidation.html)<br/>Front matter Error(s): [1](https://ci.syncfusion.com/job/Content/job/fileformat-docs/54/artifact/cireports/FrontMatterValidation/FrontMatterValidation.html)<br/>File Path Error(s): [0](https://ci.syncfusion.com/job/Content/job/fileformat-docs/54/artifact/cireports/FilePathValidation/FilePathValidation.html)<br/>CodeSnippet File Path Missing Error(s): [0](https://ci.syncfusion.com/job/Content/job/fileformat-docs/54/artifact/cireports/CodeSnippetFilePathValidation/CodeSnippetFilePathValidation.html)<br/>Toc Anchor Tags Validation Error(s): [0](https://ci.syncfusion.com/job/Content/job/fileformat-docs/54/artifact/cireports/AnchorTagsValidation/AnchorTagsValidation.html)<br/>**

AntroJames commented:
- CI failed because the below link was in unreachable state. https://opensource.adobe.com/dc-acrobat-sdk-docs/acrobatsdk/pdfs/acrobatsdk_jsdevguide.pdf
- So this failure has been occured.
 <img width="926" alt="image" src="https://github.com/syncfusion-content/fileformat-docs/assets/124988180/b0266acc-97b5-4d13-a5ac-1f6d498f07c4">

- Now, we removed old link and modified the new link in below https://opensource.adobe.com/dc-acrobat-sdk-docs/library/jsdevguide/index.html

SyncfusionBuild commented:
**Build Status: INQUEUE :clock3:<br/>Build Location: [Task_874816](https://ci.syncfusion.com/job/Content/job/fileformat-docs/55/console)**

SyncfusionBuild commented:
**Build Status: INPROGRESS :arrows_clockwise:<br/>Build Location: [Task_874816](https://ci.syncfusion.com/job/Content/job/fileformat-docs/55/console)**

SyncfusionBuild commented:
**CI Status: SUCCESS :white_check_mark:<br/>Source Compilation: SUCCESS :white_check_mark:<br/>Build Location: [Task_874816](https://ci.syncfusion.com/job/Content/job/fileformat-docs/55/)<br/>Image Alt Text Error(s): [0](https://ci.syncfusion.com/job/Content/job/fileformat-docs/55/artifact/cireports/ImageValidation/AltTextValidation.html)<br/>Image Size Error(s): [0](https://ci.syncfusion.com/job/Content/job/fileformat-docs/55/artifact/cireports/ImageValidation/ImageSizeValidation.html)<br/>Image Name Error(s): [0](https://ci.syncfusion.com/job/Content/job/fileformat-docs/55/artifact/cireports/ImageValidation/ImageNameValidation.html)<br/>FT Layout Syntax Error(s): [0](https://ci.syncfusion.com/job/Content/job/fileformat-docs/55/artifact/cireports/FTLayoutSyntaxValidation/FTStructureValidation.html)<br/>Front matter Error(s): [0](https://ci.syncfusion.com/job/Content/job/fileformat-docs/55/artifact/cireports/FrontMatterValidation/FrontMatterValidation.html)<br/>File Path Error(s): [0](https://ci.syncfusion.com/job/Content/job/fileformat-docs/55/artifact/cireports/FilePathValidation/FilePathValidation.html)<br/>CodeSnippet File Path Missing Error(s): [0](https://ci.syncfusion.com/job/Content/job/fileformat-docs/55/artifact/cireports/CodeSnippetFilePathValidation/CodeSnippetFilePathValidation.html)<br/>Toc Anchor Tags Validation Error(s): [0](https://ci.syncfusion.com/job/Content/job/fileformat-docs/55/artifact/cireports/AnchorTagsValidation/AnchorTagsValidation.html)<br/>**
